src/components/CaptureDetailDialog.js

```diff
@@ -313,6 +340,14 @@ function CaptureDetailDialog(props) {
           style={{ maxWidth: '100%' }}
           fixed
         />
+        <Link
```

Comment:
Let's remove this link from the image.

Reply:
Hey @nmcharlton, just removed that link.